models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+from uuid import uuid4
+
+class Garment(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
+    name = models.CharField(max_length=200)
+    dscription = models.TextField(blank=True)
+
```

Comment:
This is a good start for your `garment` model. Consider other properties to add in the future, like `size`, `color`, `style`, and so forth. 👍